ajax/libs/ng-dropzone/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ng-dropzone",
+  "filename": "ng-dropzone.min.js",
+  "description": "AngularJS directive for Dropzone, an easy to use drag'n'drop file upload library.",
+  "homepage": "http://www.dropzonejs.com",
+  "keywords": [
+    "angularjs",
+    "directive",
+    "drag",
+    "drop",
+    "file",
+    "upload"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/thatisuday/ng-dropzone/"
+  },
+  "license": "Apache-2.0",
+  "author": "Uday Hiwarale <uhiwarale@gmail.com> (https://github.com/thatisuday)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/thatisuday/ng-dropzone.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*!(.gz)"
```

Comment:
This fileMap's rule still fetches the `*.gz` files.
Let's modify from `"**/*!(.gz)"` to `"**/!(*.gz|*.md)"`, this will get the right files.
Thank you.